frontend/app/views/spree/shared/_mobile_navigation.html.erb

```diff
@@ -20,7 +20,12 @@
     <ul class="list-unstyled position-relative h-100 mobile-navigation-list">
       <% spree_menu.children.each do |parent| %>
           <li class="d-flex justify-content-between align-items-center mobile-navigation-list-item">
-          <%= link_to parent.name, spree_localized_item_link(parent), class: 'w-75' %>
+          <% if parent.item_type == "Container" %>
+            <%= content_tag :span, parent.name %>
+          <% else %>
+            <%= link_to parent.name, spree_localized_item_link(parent), class: 'w-75' %>
+          <% end %>
+
           <% if parent.children.present? %>
             <a class="w-25 text-right mobile-navigation-category-link" data-category="<%= parent.name.parameterize %>" href="#" aria-label="<%= Spree.t('go_to_category') %>">
```

Comment:
@MatthewKennedy should we do something about the `href="#'` here?

Reply:
Hi @damianlegawiec  That a tag with the `href="#"` was in the DOM before I made any changes on this PR. It is used for the little arrow to the mobile sub-menu as highlighted in blue in the image below.

<img width="833" alt="Screenshot 2021-05-17 at 10 15 02" src="https://user-images.githubusercontent.com/1240766/118464594-f4321680-b6f8-11eb-8850-f8f4ccb215ab.png">

If that is okay with you, I don't want to change that on this PR , I think if we were to improve that layout and code for the mobile off-canvas menu,  it should be done on it's own dedicated PR.

Reply:
I'm more than happy to change the code in a separate PR, and maybe bring that arrow off the edge of the screen, increase the height of the clickable area and such?